commit 5cf1564dd8792d7ee6db01a86aef53e4614b0daf
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-03-26T17:55:27+01:00 (7 months ago)
    
    TRITON-1354 AdminUI should use bootstrap mode for moray - binder connections

diff --git a/sapi_manifests/adminui/template b/sapi_manifests/adminui/template
index 74a02725..beae67ad 100644
--- a/sapi_manifests/adminui/template
+++ b/sapi_manifests/adminui/template
@@ -37,12 +37,9 @@
 		"{{datacenter_name}}": {
 			"rabbitmq": "{{{rabbitmq}}}",
 			"moray": {
-				"host": "{{{MORAY_SERVICE}}}",
-				"port": 2020,
-				"connectTimeout": 200,
-				"retry": {
-					"retries": 2,
-					"minTimeout": 500
+				"srvDomain": "{{{MORAY_SERVICE}}}",
+				"cueballOptions": {
+					"resolvers": [ "{{{BINDER_SERVICE}}}" ]
 				}
 			},
 			"amon": {
